resource.h

```diff
@@ -143,6 +150,55 @@ typedef struct res_net_ifstat {
 	unsigned long tx_compressed;
 } res_net_ifstat_t;
 
+typedef struct proc_info {
+	int pid;
+	char cmd[16];
+	char state;
+	int ppid;
+	int pgrp;
+	int session;
+	int tty;
+	int tpgid;
+	unsigned long flags;
+	unsigned long min_flt;
+	unsigned long cmin_flt;
+	unsigned long maj_flt;
+	unsigned long cmaj_flt;
+	unsigned long long utime;
+	unsigned long long stime;
+	unsigned long long cutime;
+	unsigned long long cstime;
+	unsigned long long start_time;
+	long priority;
+	long nice;
+	int nlwp;
+	long alarm;
+	unsigned long vsize;
+	long rss;
+	unsigned long rss_rlim;
+	unsigned long long start_code;
+	unsigned long long end_code;
+	unsigned long long start_stack;
+	unsigned long long kstk_esp;
+	unsigned long long kstk_eip;
+	unsigned long long wchan;
+	int exit_signal;
+	int processor;
+	unsigned long rtprio;
+	unsigned long policy;
+	int euid;
+	char euser[RES_USR_NAME_SZ];
+
+	/* Information from /prco/[pid]/statm */
+	unsigned long size;
+	unsigned long resident;
+	unsigned long share;
+	unsigned long text;
+	unsigned long lib;
+	unsigned long data;
+	unsigned long dt;
+} res_proc_infoall_t;
```

Comment:
Would it make sense to be even more specific about types? So instead of e.g. `unsigned long long` just use `uint64_t`, in fact you could probably use `uint64_t` also for `unsigned long`.

Reply:
makes sense, will change here and some other places.